omnibus/files/private-chef-cookbooks/private-chef/recipes/bootstrap.rb

```diff
@@ -49,18 +38,9 @@
   notifies :restart, 'service[opscode-erchef]'
 end
 
-file pivotal_key_path do
-  owner OmnibusHelper.new(node).ownership['owner']
-  group "root"
-  mode "0600"
-  content pivotal_key.to_pem.to_s
-  sensitive true
-end
-
 file OmnibusHelper.bootstrap_sentinel_file do
   owner "root"
   group "root"
   mode "0600"
-  content "You've been bootstrapped, punk. Delete me if you feel lucky. Do ya, Punk?"
+  content "You've been bootstrapped, punk. Delete me if you feel lucky. Do ya, punk?"
```

Comment:
Yeah, I've tried making it more meaningful now, including the time it was bootstrapped...